builtin/settingtypes.txt

```diff
@@ -903,6 +903,9 @@ fallback_font_shadow_alpha (Fallback font shadow alpha) int 128 0 255
 fallback_font_path (Fallback font path) filepath fonts/DroidSansFallbackFull.ttf
 
 #    Path to save screenshots at.
+#    If starting with a slash, it will be considered as an absolute path.
```

Comment:
Windows be like: `C:\path\to\minetest\`
Does not start with a slash but is an absolute path.